### ISSUE TITLE
small error in Introduction, lambdas-plutarch-level-function-terms

### ISSUE THREAD

**Issue by pb99u069** [Plutonomicon/plutarch#421]
in 

https://github.com/Plutonomicon/plutarch/blob/master/docs/Introduction/Plutarch%20Terms/Plutarch%20Lambdas.md#lambdas-plutarch-level-function-terms

'Plutarch-level `id` lambda' is stated as:
```
pid :: Term s (a :--> a)
pid = plam x
```

but should be:

```
pid :: Term s (a :--> a)
pid = plam termId
```

**Comment by TotallyNotChase** [comment]
Good catch! Would you like to make a PR?

**Comment by L-as** [comment]
`plam id` is also correct

**Comment by TotallyNotChase** [comment]
fixed via #422
